AWSSDKSwiftCLI/Package.swift

```diff
@@ -27,31 +29,6 @@ let package = Package(
                 .process("Resources/DocIndex.Base.md")
             ]
         ),
-        .executableTarget(
-            name: "spr-publish",
-            dependencies: [
-                "SPR",
-                "AWSCLIUtils",
-                .product(name: "ArgumentParser", package: "swift-argument-parser"),
-            ]
-        ),
-        .executableTarget(
-            name: "spr-multi-publish",
-            dependencies: [
-                "SPR",
-                "AWSCLIUtils",
-                "spr-publish",
-            ]
-        ),
-        .target(
-            name: "SPR",
-            dependencies: [
-                "AWSCLIUtils",
-                .product(name: "ArgumentParser", package: "swift-argument-parser"),
-                .product(name: "AWSS3", package: "aws-sdk-swift"),
-                .product(name: "AWSCloudFront", package: "aws-sdk-swift"),
-            ]
-        ),
```

Comment:
All the SPR targets in `AWSSDKSwiftCLI`, along with the SDK dependency, are moved to the new SPRCLI package defined below.